playground/infrastructure/helper.py

```diff
@@ -0,0 +1,229 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import os
+import yaml
+
+from dataclasses import dataclass
+from typing import List
+from api.v1.api_pb2 import Sdk, SDK_JAVA, SDK_UNSPECIFIED, STATUS_UNSPECIFIED, SDK_GO, SDK_PYTHON, STATUS_VALIDATING, \
+    STATUS_PREPARING, STATUS_COMPILING, STATUS_EXECUTING
+from grpc_client import GRPCClient
+
+SUPPORTED_SDK = {'java': SDK_JAVA, 'go': SDK_GO, 'py': SDK_PYTHON}
+END_OF_LICENCE = "limitations under the License."
+START_OF_IMPORT = "\nimport "
+START_OF_FROM = "\nfrom "
+BEAM_PLAYGROUND = "Beam-playground"
+PAUSE_DELAY = 10
```

Comment:
can we move these constants to config.py?

Reply:
I think that these constants need for the `helper` only. Moreover, they don't configure anything.